Model/src/BoardPackage/RailRoadSquare.java

```diff
@@ -1,7 +1,131 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
-public class RailRoadSquare {
+/*****************************************************************
+ The RailRoadSquare class is a BoardSquare that tracks
+ information specific to RailRoadSquares.
+
+ @author Dylan Kernohan
+ @version 2/12/2018
+ *****************************************************************/
+public class RailRoadSquare extends BoardSquare{
+
+    /**The price of the RailRoad square*/
+    private final int PRICE;
+
+    /**The initial rent of the RailRoad square*/
+    private final int BASE_RENT;
+
+    /**The rent if a player owns two RailRoads*/
+    private final int TWO_RENT;
+
+    /**The rent if a player owns three RailRoads*/
+    private final int THREE_RENT;
+
+    /**The rent if a player owns four RailRoads*/
+    private final int FOUR_RENT;
+
+    /**The value a player receives if they mortgage this RailRoad*/
+    private final int MORTGAGE_VALUE;
+
+    /**A flag to tell if this RailRoad is currently mortgaged*/
+    private boolean isMortgaged;
+
+
```

Comment:
Need an owner variable. We forgot that in the UML. Add related getter and setter.

Reply:
Ah, yes. good catch. Will get to that in a bit.